mexico-city/src/mc_nitro.rs

```diff
@@ -0,0 +1,219 @@
+//! AWS Nitro-Enclaves-specific material for the Mexico City enclave
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Licensing and copyright notice
+//!
+//! See the `LICENSE.markdown` file in the Veracruz root directory for
+//! information on licensing and copyright.
+
+use byteorder::{ByteOrder, LittleEndian};
+use nix::sys::socket::listen as listen_vsock;
```

Comment:
Can you run `cargo fmt` on the code to e.g. group together related imports from `std::os::` and similar.  That should also fix some issues with indentation in `nitro_main` below.